client/gutenberg/extensions/simple-payments/edit.js

```diff
@@ -3,34 +3,302 @@
 /**
  * External dependencies
  */
+import get from 'lodash/get';
+import { __ } from '@wordpress/i18n';
```

Comment:
Noting that once #28088 lands, we'll have to use our own i18n wrappers.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/28065/commits/d49d93ca8fdfb6c61189a49d8d3842e8b9e519d5